tools/config-generator/k8s_testgrid_config.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// data definitions that are used for the config file generation of k8s testgrid
+
+package main
+
+import (
+	"sort"
+)
+
+const (
+	k8sTestgridTempl = "k8s_testgrid.yaml"
+)
+
+type k8sTestgridData struct {
+	NamedDashboards          []string
+	KnativeDashboards        []string
```

Comment:
This will cause inconsistency when we add/remove an org from `config_knative.yaml`. Can we read `Knative`, `KnativeSandbox` and `Google` from the org names in `config_knative.yaml`, instead of naming them here?

Reply:
Done, thanks.